maven-plugin/pom.xml

```diff
@@ -68,6 +68,7 @@
     <dependency>
       <groupId>xml-apis</groupId>
       <artifactId>xml-apis</artifactId>
+      <scope>provided</scope>
```

Comment:
I looked into this, and adding an exclusion in the `dependencyManagement` of `hadoop-client` and `hadoop-minicluster` in the root pom for `xerces:xercesImpl` is sufficient.

These are being brought in transitively because this version of hadoop (2.6.4) is still using *very* old versions of (mortbay) jetty-util.